stdlib/public/Concurrency/TaskSleepDuration.swift

```diff
@@ -142,4 +142,20 @@ extension Task where Success == Never, Failure == Never {
   ) async throws {
     try await clock.sleep(until: deadline, tolerance: tolerance)
   }
+  
+  /// Suspends the current task for the given duration on a continuous clock.
+  ///
+  /// If the task is canceled before the time ends, this function throws 
```

Comment:
API reference and documentation follow different spelling for this word.  In API reference, we spell it "cancelled" to follow Cocoa API naming guidelines, which maintain continuity going back to NeXT.  In documentation, we spell it "canceled" following Apple Style Guide.

```suggestion
  /// If the task is cancelled before the time ends, this function throws 
```

Reply:
@amartini51 I copied and pasted the documentation for `Task.sleep(until:...)`, so this spelling and the other changes requested are problems in the existing docs as well. Maybe a follow-up PR that addresses all of the documentation is in order instead?

FWIW there are 50/300 instances of a single L "canceled" in the repo:

``` console
$ git grep "canceled" | wc -l
      51
$ git grep "cancelled" | wc -l
     258

```

Reply:
Do you mind spelling it correctly here, and I'll make a separate PR to fix that one?  I've been cleaning them up as I find them.

Doing a corpus search via `git grep` won't give you quite the right picture, because of the two different contexts.  You need something more like this: